project/Dependencies.scala

```diff
@@ -65,33 +65,33 @@ object Dependencies {
     commonDeps ++ jacksonOverrides ++ Seq(
       evolutions,
       jdbc,
-      "com.gu.play-googleauth" %% "play-v30" % "10.0.1",
-      "com.gu.play-secret-rotation" %% "play-v30" % "8.4.5",
-      "com.gu.play-secret-rotation" %% "aws-parameterstore-sdk-v2" % "8.4.5",
+      "com.gu.play-googleauth" %% "play-v30" % "18.0.0",
+      "com.gu.play-secret-rotation" %% "play-v30" % "13.1.2",
+      "com.gu.play-secret-rotation" %% "aws-parameterstore-sdk-v2" % "13.1.2",
       "org.pegdown" % "pegdown" % "1.6.0",
       "com.adrianhurt" %% "play-bootstrap" % "1.6.1-P28-B3", // scala-steward:off,
-      "org.scanamo" %% "scanamo" % "1.1.1",
+      "org.scanamo" %% "scanamo" % "3.0.0",
       "software.amazon.awssdk" % "dynamodb" % Versions.aws,
       "software.amazon.awssdk" % "sns" % Versions.aws,
-      "org.quartz-scheduler" % "quartz" % "2.3.2",
+      "org.quartz-scheduler" % "quartz" % "2.5.0",
```

Comment:
This version is currently prevents the server from starting - https://github.com/guardian/riff-raff/pull/1408.